app/package.json

```diff
@@ -35,6 +35,7 @@
     "express-session": "^1.17.2",
     "graphql-upload": "^15.0.2",
     "js-cookie": "^3.0.1",
+    "json-schema": "^0.4.0",
```

Comment:
Should we be using from the [types](https://www.npmjs.com/package/@types/json-schema) package or does it not really matter?

Reply:
Ah gotcha, cool! I did check that the `@types /json-schema` supports draft-07 since I did notice that the summary randomly cuts off before listing anything after 6 and it is in the files for the package.